File: core/ParamsAndDtos/OrderItemsAndAgentsToFwdDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace core.ParamsAndDtos
{
    public class OrderItemsAndAgentsToFwdDto
    {
        public ICollection<OrderItemToFwdDto> Items {get; set;}
        public ICollection<AssociateToFwdDto> Agents {get; set;}
        public DateTime DateForwarded {get; set;}
    }
}

File: client/src/app/shared/models/orderItemsAndAgentsToFwdDto.ts

import { ICustomerOfficialDto } from "./customerOfficialDto";
import { IOrderItemToFwdDto } from "./orderItemToFwdDto";

export interface IOrderItemsAndAgentsToFwdDto
{
     items: IOrderItemToFwdDto[];
     agents: ICustomerOfficialDto[];
     dateForwarded: Date;
}

export class OrderItemsAndAgentsToFwdDto implements IOrderItemsAndAgentsToFwdDto
{
     items: IOrderItemToFwdDto[];
     agents: ICustomerOfficialDto[];
     dateForwarded: Date;
}